AndroidTool/ApkHandler.swift

```diff
@@ -8,29 +8,29 @@
 
 import Cocoa
 
-protocol ApkHandlerDelegate {
+protocol ApkHandlerDelegate: AnyObject {
```

Comment:
what does this enable?

Reply:
1. it allows `delegate` to be `weak`
2. protocols that will never be conformed to by structs can get some performance and binary size benefits by making this explicit, so that the compiler doesn't have to add code to box them.

Reply:
whats a case where this delegate protocol will be boxed?

Reply:
Any struct that's too big to just copy. If memory serves at WWDC they suggest if your struct has 3+ fields, it'll probably get boxed when you try to pass it in a place where a protocol is taken.